src/transaction/components/Transaction.tsx

```diff
@@ -35,7 +34,6 @@ export function Transaction({
       calls={calls}
       capabilities={capabilities}
       chainId={accountChainId}
-      contracts={contracts}
```

Comment:
i'm leaning towards just removing it to avoid confusion. i.e devs might think they need to pass calls to calls and contracts to contracts which would cause the order of execution to be unreliable. plus it is a super small change on the developer side (just changing prop name to calls).

Reply:
If you left the calls and contracts error in place we could mark contracts as deprecated for a few releases before removing.  it would at least give the devs a heads up over the next few releases before the forcing them to update due to a breaking change.

Reply:
curious what the rest of the team thinks but personally still leaning towards just removing it since it is such a small change on the dev side.

Reply:
+1 on marking as deprecated for a few releases, then removing. It should be a simple change for the dev but nice to give them a bit of time to switch over

Reply:
Actually, reconsidered this. Given it's a simple prop name change, I'd probably lean towards just making the breaking change now